regression/cbmc/r_w_ok_inconsistent_valid/test-global-no-cp.desc

```diff
@@ -0,0 +1,11 @@
+KNOWNBUG
+main.c
+--function test_global_pointer --pointer-check --no-simplify --no-propagation
+^EXIT=0$
```

Comment:
When this works it should be exit=10 right? I'd right the test correct so that someone doesn't think it works and turns it on

Reply:
No when it works it should be `EXIT=0`.